package.json

```diff
@@ -21,7 +21,7 @@
     "build:all": "yarn build:packages && yarn build:refapps && yarn build:docker",
     "build:refapps": "LOCAL_PACKAGES=true lerna run build:refapps",
     "build:packages": "LOCAL_PACKAGES=true lerna run build",
-    "build:all-packages": "LOCAL_PACKAGES=true lerna run build && LOCAL_PACKAGES=true lerna run build:refapps",
+    "build:all-packages": "LOCAL_PACKAGES=true lerna run build && LOCAL_PACKAGES=true lerna run build:refapps && lerna run build-python",
```

Comment:
i suggest to build first, @scramjet/adapters depends on it

Reply:
I've renamed the build step to `build`, so it shouldn't be necessary anymore.p